src/pages/Dashboard/Dashboard.tsx

```
@@ -1,6 +1,6 @@
 // import { Table } from "../../components/Table"
 
-import { FaCircle, FaSyncAlt } from "react-icons/fa";
+import { FaCircle, FaSyncAlt, FaTimes } from "react-icons/fa";
```

ESLint

[error] 'FaTimes' is defined but never used.

(@typescript-eslint/no-unused-vars)